tests/Data/Text/Orphans.hs

```diff
@@ -14,3 +15,6 @@ instance Arbitrary CaseSensitivity where
 -- TODO: Slow placeholder implementation until we can use text-2.0
 instance Arbitrary Utf8.Text where
   arbitrary = fmap Utf8.pack arbitrary
+
+instance IsString Utf8.Text where
+    fromString = Utf8.pack
```

Comment:
We don't need to make this an orphan instance, I think. Sounds generally useful.

Reply:
I'll address this in the coming cleanup PR for the `Utf8` module.